modules/smaatoBidAdapter.js

```diff
@@ -113,9 +119,44 @@ export const spec = {
    * @return boolean True if this is a valid bid, and false otherwise.
    */
   isBidRequestValid: (bid) => {
-    return typeof bid.params === 'object' &&
-      typeof bid.params.publisherId === 'string' &&
-      typeof bid.params.adspaceId === 'string';
+    if (typeof bid.params !== 'object') {
+      utils.logError('[SMAATO] Missing params object');
+      return false;
+    }
+
+    if (typeof bid.params.publisherId !== 'string') {
```

Comment:
that would be so cool if we can validate that it is string parsable into integer, smth like [that](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger)
it would greatly reduce the amount of wrongly set up configs

Reply:
I don't even know why the setting is a string instead of an int :see_no_evil:

Reply:
yeah, me might wanna introduce a "breaking change" at some point :D